### DISCUSSION TITLE
Use of AMD hardware, and using Linux as base OS environment.

### DISCUSSION THREAD

**Discussion by Dainah-0xdeadc0de** [KoljaB/Linguflex#13] [Ideas]
Thanks for the hard work on the project and contributions to our AI overlords. 

On a more serious note, I'd like to help in some form, I can volunteer my hardware as a testbed for Linux / AMD changes.

(I'm not a coder but can help bug test)
I'd like to help anyway I can to the project to be more OS independent, specifically with:
- Use of AMD hardware using SCALE / ZLUDA, to circumvent CUDA/Nvidia GPU requirement.
- Easier Use of this repo project (and your others) on Linux, convert bat files to bash script equivalents.

Is there a possibility this can be done in the future or will the project be limited to Windows for the time being?

**Comment by KoljaB** [comment]
Support for this would be awesome. Bringing Linguflex to Linux is absolutely one of my main priorities currently. I just installed Linux some days ago exactly to do this. 

My approach is to work from the bottom up. I'll test and adapt each of Linguflex's low-level dependencies on Linux until they work correctly. Once that's done, I'll run the entire system and hope to be left with only minor issues at that point.

I've made some progress with the speech-to-text and text-to-speech components. They already work, yet still some minor issues left. Will fix these this week and release updated versions of RealtimeSTT and RealtimeTTS with Linux-specific fixes. 

It would be fantastic if you could test these on your setup.

Getting these two libraries working smoothly on Linux would already represent a great portion of the work needed to bring Linguflex to the platform. I really appreciate your offer to help.
